src/mahoji/lib/abstracted_commands/duelCommand.ts

```diff
@@ -72,6 +73,21 @@ export async function duelCommand(
 		return "Duel cancelled, user didn't accept in time.";
 	}
 
+	function duelFail(err: any) {
+		duelMessage.delete().catch(noOp);
+		const wagerAmount = amount ?? 0;
+		const extras = {
```

Comment:
This extra data is required so we can properly  track/refund a duel that fails, as that has already happened.

The function is required to avoid duplicate code